cli/src/commands/upload.ts

```diff
@@ -50,6 +51,18 @@ export const uploadCommand = new Command('upload')
       const address = globalOptions.address
       console.log(`Editing website at address ${address}, no deploy needed`)
 
+      const isimmutable = await isImmutable(
+        address,
+        globalOptions.node_url,
+        true
+      )
+      if (isimmutable) {
+        console.error(
+          `The website at address ${address} is immutable. It cannot be edited anymore.`
+        )
+        process.exit(1)
+      }
```

Comment:
I'm thinking that we could make that a util function for the commands because we might need it somewhere else, such as the `delete` command